doc/api/next_api_changes/behavior/15127-TAC.rst

```diff
@@ -0,0 +1,8 @@
+Raise or warn on registering a colormap twice
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When using `matplotlib.cm.register_cmap` to register a user provided
+or third-party color map it will now raise a `ValueError` if trying to
+over-write one of the built in color maps and warn if trying to over
```

Comment:
Sorry, just found 3 more instances of "color map" in this file that I missed before.

Reply:
one more try!